main.css

```diff
@@ -0,0 +1,769 @@
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.creative-body {
+  background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.8)), url(./TNW-Images/creative.jpg);
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.podium-body {
+  background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.8)), url(./TNW-Images/podium.png);
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.insights-body {
+  background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.8)), url(./TNW-Images/insights.jpg);
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.launch-body {
+  background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.8)), url(./TNW-Images/launch.jpg);
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.distract-body {
+  background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.8)), url(./TNW-Images/distract.jpg);
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.common-text {
+  padding: 12em 0 1em 1em;
+  margin-top: 1em;
+}
+
+.common-text h5 {
+  color: white;
+  font-size: 0.8em;
+}
+
+.common-text p {
+  color: #999;
+  font-size: 0.7em;
+}
+
+.list-grid-mid {
+  display: flex;
+  justify-content: space-between;
+  background-color: #ffebe8;
+  padding-left: 1em;
+}
+
+.list-grid-mid h5 {
+  color: #f42;
+  padding-top: 1em;
+}
+
+.list-grid-mid img {
+  width: 96px;
+  height: 96px;
+}
+
+.list-grid-footer {
+  padding-left: 1em;
+}
+
+.list-grid-footer li {
+  line-height: 25px;
+  color: #f42;
+}
+
+footer {
+  width: 100%;
+}
+
+.first-footer {
+  background-color: #333;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr;
+  justify-items: center;
+  padding: 3em 0;
+}
+
+.social-icons {
+  display: flex;
+  list-style: none;
+  padding: 1em 0 0 auto;
+}
+
+.social-icons li {
+  padding: 0 0.5em;
+}
+
+.social-list {
+  display: flex;
+  list-style: none;
+  padding: 1em 0;
+}
+
+.second-footer {
+  background-color: black;
+  text-align: center;
+  padding: 2em 0;
+}
+
+.second-footer p {
+  color: #999;
+  font-size: 0.9em;
+}
+
+.second-footer span {
+  color: red;
+  font-size: 1.5em;
+}
+
+.bar-one a:hover {
+  color: white;
+}
+
+.bar-two a:hover,
+.funds-address a:hover,
+.grid-address a:hover,
+.news p a:hover,
+.list-grid h2 a:hover {
+  color: black;
+}
+
+.list-2 button:hover {
+  background-color: #609;
+  cursor: pointer;
+}
+
+@media only screen and (max-width: 1024px) and (min-width: 601px) {
+  .removable-img {
+    display: block;
+  }
+
+  .bar-two {
+    justify-content: space-between;
+  }
+
+  .bar-two .list-2 {
+    display: none;
+  }
+
+  .bar-one {
+    display: none;
+  }
```

Comment:
As requested by the previous TSE, please use the DRY principle to optimize your CSS file